pinot-common/src/thrift/request.thrift

```diff
@@ -154,6 +154,7 @@ struct BrokerRequest {
  15: optional HavingFilterQuery havingFilterQuery;
  16: optional HavingFilterQueryMap havingFilterSubQueryMap;
  17: optional query.PinotQuery pinotQuery;
+ 18: optional list<SelectionSort> orderBy;
```

Comment:
is it possible to rename SelectionSort to OrderBy?

Reply:
Yes, I thought about that as well. It might cause backward compatibility issues. Given that we are going to move to PinotQuery and drop BrokerReuqest this won't be an issue for long, so I decided it leave it as-is.

Reply:
Makes sense.